src/Examples/Flowsheets/HDA_flowsheet_with_costing.ipynb

```diff
@@ -0,0 +1,1598 @@
+    "- def cost_vertical_vessel(blk, material_type=VesselMaterial.CarbonSteel, shell_thickness=1.25 * pyo.units.inch, weight_limit=1, aspect_ratio_range=1, include_platforms_ladders=True, vessel_diameter=None, vessel_length=None, number_of_units=1, number_of_trays=None, tray_material=TrayMaterial.CarbonSteel, tray_type=TrayType.Sieve)\n",
+    "- def cost_horizontal_vessel(blk, material_type=VesselMaterial.CarbonSteel, shell_thickness=1.25 * pyo.units.inch, include_platforms_ladders=True, vessel_diameter=None, vessel_length=None, number_of_units=1)\n",
+    "- def cost_fired_heater(blk, heat_source=HeaterSource.Fuel, material_type=HeaterMaterial.CarbonSteel, integer=True)\n",
+    "- def cost_compressor(blk, compressor_type=CompressorType.Centrifugal, drive_type=CompressorDriveType.ElectricMotor, material_type=CompressorMaterial.StainlessSteel, integer=True)\n",
+    "- def cost_fan(blk, fan_type=FanType.CentrifugalBackward, fan_head_factor=1.45, material_type=FanMaterial.StainlessSteel, integer=True)\n",
+    "- def cost_blower(blk, blower_type=BlowerType.Centrifugal, material_type=BlowerMaterial.StainlessSteel, integer=True)\n",
+    "- def cost_turbine(blk, integer=True)\n",
+    "- def cost_pump(blk, pump_type=PumpType.Centrifugal, material_type=PumpMaterial.StainlessSteel, pump_type_factor=1.4, motor_type=PumpMotorType.Open, integer=True)\n",
+    "- def cost_pressure_changer(blk, mover_type=\"compressor\", **kwargs)\n",
+    "\n",
+    "As a convenience for users, common unit models are mapped with appropriate costing methods. Therefore, for quick costing with defaults, users may import the `unit_mapping` dictionary and pass the unit model type as below:\n",
+    "\n",
+    "*unit_mapping = {  \n",
+    "        CSTR: cost_vertical_vessel,  \n",
+    "        Compressor: cost_compressor,  \n",
+    "        Flash: cost_vertical_vessel,  \n",
+    "        Heater: cost_fired_heater,  \n",
+    "        HeatExchanger: cost_heat_exchanger,  \n",
+    "        HeatExchangerNTU: cost_heat_exchanger,  \n",
+    "        PFR: cost_horizontal_vessel,  \n",
+    "        PressureChanger: cost_pressure_changer,  \n",
+    "        Pump: cost_pump,  \n",
+    "        StoichiometricReactor: cost_horizontal_vessel,  \n",
+    "        Turbine: cost_turbine,  \n",
+    "    }*\n",
+    "\n",
+    "*costing_blk = unit_mapping[type(blk)]*\n",
+    "\n",
+    "The mapping is inheritance aware, e.g. pumps will try costing with cost_pump() and fall back to cost_pressure_changer."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Add Operating Cost Equations\n",
+    "Before adding capital costing blocks, we will add operating cost equations taken from the prior examples. The examples assume constant cooling and heating coefficients over an annual cost basis. The IDAES Generic Costing Framework does not currently support variable cost calculations."
```

Comment:
I thinki you need to clarify what you mean by "prior examples" - You should at least provide the link to those.